index.html

```diff
@@ -2035,9 +2030,36 @@ <h3>Securing Mechanisms</h3>
 mutually exclusive.
         </p>
         <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be secured by at least
-one securing method.
+<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be
+secured by at least one securing mechanism.
+        </p>
+        <p>
+Securing mechanism specifications:
         </p>
+
+        <ul>
+          <li>
+MUST document normative algorithms that can be used to secure and verify
+<a>verifiable credentials</a> and <a>verifiable presentations</a>.
+          </li>
+          <li>
+MUST provide a verification mechanism that returns only the information in the
+<a>conforming document</a> that has been secured. It MAY provide an
+additional mechanism to receive other information that might be helpful
+to developers. The concrete interface that is
+expected to be provided is detailed in the
+<a href="#verify-securing-mechanism">securing mechanism verification
+algorithm</a> section.
+          </li>
+          <li>
+SHOULD provide integrity protection for any information referenced by a URL that
+is critical to validation. Mechanisms that can achieve this protection are
+discussed in Section <a href="#integrity-of-related-resources"></a> and Section
+<a href="#base-context"></a>.
+          </li>
+        </ul>
```

Comment:
```suggestion
          <li>
MUST provide integrity protection for the <a>conforming document</a>.
          </li>
          <li>
MUST verify the authorship of the <a>verifiable credential</a> or <a>verifiable presentation</a>.
          </li>          
        </ul>        
```

Reply:
These two relate to:
> A verifiable credential is a tamper-evident credential that has authorship that can be cryptographically verified.

or
> A [verifiable credential](https://w3c.github.io/vc-data-model/#dfn-verifiable-credential) is a set of tamper-evident [claims](https://w3c.github.io/vc-data-model/#dfn-claims) and metadata that cryptographically prove who issued it.

Reply:
IMO, it would be valuable that the verifier knows the `issuer` issued the VC, or the `holder` presented the VP after running the securing mechanism verification algorithm.

Reply:
I've raised issue #1386 to track this concern.

I took the suggestion for the first bullet item in commit 5e767d9f44f341ccd423b6a2e99268a493db3e9d.